content/en/docs/installing-on-linux/cluster-operation/add-edge-nodes.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Add Edge Nodes"
+keywords: 'Kubernetes, KubeSphere, KubeEdge'
+description: 'Add edge nodes to your cluster.'
+linkTitle: "Add Edge Nodes"
+weight: 3630
+---
+
+KubeSphere leverages [KubeEdge](https://kubeedge.io/en/), to extend native containerized application orchestration capabilities to hosts at edge. With separate cloud and edge core modules, KubeEdge provides complete edge computing solutions while the installation may be complex and difficult. 
+
+![kubeedge_arch](/images/docs/installing-on-linux/add-and-delete-nodes/add-edge-nodes/kubeedge_arch.png)
+
+{{< notice note >}}
+
+For more information about different components of KubeEdge, see [the KubeEdge documentation](https://docs.kubeedge.io/en/docs/kubeedge/#components).
+
+{{</ notice >}} 
+
+After an edge node joins your cluster, the native KubeEdge cloud component requires you to manually configure iptables so that you can use commands such as `kubectl logs` and `kubectl exec`. In this connection, KubeSphere features an efficient and convenient way to add edge nodes to a Kubernetes cluster. It uses supporting components (for example, EdgeWatcher and IPTables-Operator) to automatically configure iptables.
+
+![edge-watcher](/images/docs/installing-on-linux/add-and-delete-nodes/add-edge-nodes/edge-watcher.png)
+
+This tutorial demonstrates how to add an edge node to your cluster.
+
+## Prerequisites
+
+- You have enabled [KubeEdge](../../../pluggable-components/kubeedge/).
+- You have an available node to serve as an edge node with [Docker](https://docs.docker.com/engine/install/) 19.03.12 or later installed. The node can run either Ubuntu (recommended) or CentOS. This tutorial uses Ubuntu 18.04 as an example.
+- Edge nodes, unlike Kubernetes cluster nodes, should work in a separate network.
+
+## Configure an Edge Node
+
```

Comment:
To config an edge node, 2 steps needed.
Configuring edge mesh should be the second step.
The first step should be installing container runtime：

KubeEdge doc mentioned the following container runtime are supported：
Docker
Containerd
Cri-o
Virtlet

These official doc links could be added to our doc as well
https://docs.kubeedge.io/en/docs/
https://docs.kubeedge.io/en/docs/advanced/cri/

Reply:
Added.

Reply:
Changed.